js/utilities.js

```diff
@@ -836,6 +836,57 @@
         };
     };
 
+    String.prototype.transformToDisplay = function() {
+      return this.split('_').
+            splice(1).map(function(word)
+              {
+                return word[0].toUpperCase() + word.slice(1);
+              }).join(' ');
+    };
+    /**
+     * @function
+     * @param  {string} errorStatusText    http error status text
+     * @param  {string} generatedErrMessage response data returned by http request
+     * @return {object}                    error object
+     */
+    module._conflictErrorMapping = function(errorStatusText, generatedErrMessage) {
```

Comment:
As karl mentioned in one of the meetings, these error messages might change.  Can you change some of the logics that if they are not as what you are expecting, they still produce meaningful messages?

Also add the expected format for each of those as a comment. So we can refer to that.

Reply:
Made few changes so that in case of string literals not showing p; we can still have useful messages. However we are still reliant on message thrown for error processing.